AutoT4/ExtensionMethods.cs

```diff
@@ -8,6 +8,8 @@ namespace BennorMcCarthy.AutoT4
 {
     public static class DTEExtensions
     {
+        private const string SolutionFolder = "{66A26720-8FB5-11D2-AA7E-00C04F688DDE}";
```

Comment:
Yes but it would require EnvDTE80. Since I did not want to add visual studio version requirement, I have put it in a constant. Then it will work with all visual studio versions.